integration-tests/common/staking.ts

```diff
@@ -23,6 +23,16 @@ export interface RPCStakedState {
 }
 /* eslint-enable camelcase */
 
+/* eslint-disable camelcase */
+export interface FlattenStakedStake {
+    nonce: BigNumber;
+    bonded: BigNumber;
+    unbonded: BigNumber;
+    unbondedFrom: number;
+    address: string;
+}
+/* eslint-enable camelcase */
```

Comment:
https://eslint.org/docs/rules/camelcase
I think the closest would be not to apply it to `properties`. However, I prefer to enable it globally because camelcase is not prefer in JS conventions. The reason we used it is because of Tendermint RPC response format.